### ISSUE TITLE
Feature Request: Fallback redirect targets

### ISSUE THREAD

**Issue by limaceous-bushwhacker** [einaregilsson/Redirector#377]
Something similar was mentioned 2 years back [here](https://github.com/einaregilsson/Redirector/issues/272). Wonder if there's enough demand now to get someone to work on this?

Btw is there project active again? Saw some recent activity here. It's an amazing project (thank you and RIP einaregilsson) that I've continued to use for a year now.

**Comment by Gitoffthelawn** [comment]
Yes, the project is active, but we need volunteers, especially for JavaScript coding.  Are you able to offer help?

**Comment by limaceous-bushwhacker** [comment]
> Yes, the project is active, but we need volunteers, especially for JavaScript coding. Are you able to offer help?

That's good to hear. I'm afraid I can't help with the coding part.

**Comment by Gitoffthelawn** [comment]
No problem.  Thanks for writing back.

**Comment by chabala** [comment]
Did this get completed?

**Comment by Gitoffthelawn** [comment]
> Did this get completed?

No, it did not.  Are you interested in implementing it?

**Comment by chabala** [comment]
That's probably outside my skill, don't do much JavaScript. I was just confused by the 'closed as completed', but no linked PR.
